flycheck.el

```diff
@@ -10979,6 +10979,10 @@ This is either a parent directory containing a Gemfile, or nil."
    buffer-file-name
    (locate-dominating-file buffer-file-name "Gemfile")))
 
+(defun flycheck-ruby--filter-errors (errors)
+  "Filter Rubocop ERRORS attributed to dummy stdin filename."
+  (flycheck-remove-error-file-names (concat default-directory "stdin") errors))
```

Comment:
You want expand-file-name, not concat, I think

Reply:
Thanks, yes, that's better. Done in https://github.com/flycheck/flycheck/pull/1793/commits/1694a6e4595accd423f0a8c5d995401fd2cf163c